_includes/desktop-builders-setting.md

```diff
@@ -0,0 +1,60 @@
+If you have turned on the Docker Desktop
+[**Builds** view (beta)](/desktop/use-desktop/builds/), you can use the
+**Builders** tab to inspect and manage builders in the Docker Desktop settings.
+
+### Inspect
+
+To inspect builders, find the builder that you want to inspect and select the
+expand icon. You can only inspect active builders.
+
+The detailed builder view shows:
+
+- Builder status
+- Information about the BuildKit runtime
+- Capabilities and driver type
+- Supported platforms
+- Disk usage
+
+### Select a different builder
+
+The **Selected builder** section displays the selected builder.
+To select a different builder:
+
+1. Find the builder that you want to use under **Available builders**
+2. Open the drop-down menu next to the builder's name.
+3. Select **Use** to switch to this builder.
+
+Your build commands now use the selected builder by default.
+
+### Create a builder
+
+To create a builder, use the Docker CLI. See
+[Create a new builder](/build/builders/manage/#create-a-new-builder)
+
+### Remove a builder
+
+You can remove a builder if:
+
+- The builder isn't your [selected builder](/build/builders/#selected-builder)
+- The builder isn't a [default builder](/build/builders/#default-builder),
+  associated with a Docker context.
+
+To remove a builder:
+
+1. Find the builder that you want to remove under **Available builders**
+2. Open the drop-down menu.
+3. Select **Remove** to remove this builder.
+
+If the builder uses the `docker-container` or `kubernetes` driver,
+the build cache is also removed, along with the builder.
```

Comment:
That's true. Not related but we could have a modal on deletion to confirm deletion and a checkbox if we want to keep the BuildKit state or not.

Reply:
@nico1510 We don't have a confirmation modal on deletion atm. I think it would a neat addition.